core/services/pg/utils.go

```diff
@@ -39,6 +39,8 @@ func init() {
 var (
 	// DefaultQueryTimeout is a reasonable upper bound for how long a SQL query should take
 	DefaultQueryTimeout = 10 * time.Second
+	// OneMinuteQueryTimeout is a bigger upper bound for how long a SQL query should take
+	OneMinuteQueryTimeout = 1 * time.Minute
```

Comment:
Maybe rename this (and the func) to something like `SlowQueryTimeout` or `LongQueryTimeout`? Otherwise, eliminate it and just set `time.Minute` directly from within `WithOneMinuteTimeout()`.